tests/questions/shortFirstEmbeddedHTML.gift

```diff
@@ -0,0 +1,3 @@
+[html]{ = Asia } is the largest continent in the world.
```

Comment:
This no longer parses with the latest grammar (I fixed some other bugs with `->` and `\:` in titles). Anyway, I think this notation doesn't add anything. Can you remove this test. I will accept the changes after that.

Reply:
I'll remove the test in the next push, however I believe there's no other way of setting the question type to html or markdown without the tag in the front of the question, other than writing it with the five underlines. If I find a solution however, I'll be sure to do another pull request.